wpilib-sys/wpilib-sys.mk

```diff
@@ -6,6 +6,20 @@
 # except according to those terms.
 
 local_dir := $(dir $(lastword $(MAKEFILE_LIST)))
+toolchain := $$CXX_FRC
```

Comment:
Is this correct, or should be `$(CXX_FRC)`?

Reply:
Sadly, thats correct. Makefiles have weird syntax and work with their own variable system.

If I were to try referencing an environment variable the regular way, it would start parsing and give up when it realizes there are no makefile variable that could fit. So if I had defined `CXX2` before hand, it would give up half way through and delete the characters up to the point where it failed. The resulting line would come to `toolchain := _FRC`. A second `$` acts as an escape character and allows it to be read like any other shell command. Thats also why I included two on line 20.

Reply:
Oh, right. I think the better way to deal with this would be to use `?=`.